assets/scripts/cwp-notify-categories.js

```diff
@@ -8,6 +8,11 @@
 
     $(document).ready(function () {
 
+        // Handles the select all functionality
+        $('#select_all').click(function() {
+            $(':checkbox').prop('checked', this.checked);
```

Comment:
this doesn't work. 
change to `$('.notifiable-categories').prop('checked', this.checked);` and it works

Reply:
In chrome you'll have to clear cache, then it should work. If not let me know.

Reply:
without specifying a class that belongs to the checkbox we're targeting you hit all checkboxes on the page (including ones we don't want to target)

Reply:
oh, I thought you meant it didn't work at all. Okay, that makes perfect sense I'll update it.